lua/nvim_aider/init.lua

```diff
@@ -96,6 +96,9 @@ function M.setup(opts)
       M.terminal.command(commands.drop.value, relative_filepath)
     end
   end, {})
+
+  -- Setup nvim-tree integration
+  require("nvim_aider.tree").setup()
```

Comment:
needs to be optional?

Reply:
did you see the change proposal i send you https://github.com/devenv/nvim-aider/pull/1 ?

[There](https://github.com/devenv/nvim-aider/blob/d4db314152d50128d839c9ed9f2d219b62dfb6e8/lua/nvim_aider/tree.lua#L59) is a check if nvim-tree is installed, therefore I think this is fine